src/services/textChanges.ts

```diff
@@ -988,7 +988,6 @@ export class ChangeTracker {
     public insertNodeInListAfter(sourceFile: SourceFile, after: Node, newNode: Node, containingList = formatting.SmartIndenter.getContainingList(after, sourceFile)): void {
         if (!containingList) {
             Debug.fail("node is not a list element");
-            return;

```

Comment:
How does this change relate to the issue being fixed?

Reply:
@RyanCavanaugh good question. You see, `Debug.fail` return type is `never`, the code in the block below is unreachable. Also I have fix all code action on save enabled and for some reason, TypeScript integration in VS Code decides to apply all unreachable code quick fixes on save... I was too focused on thinking of how to make other tests happy, so didn't notice this change in diff. Will definitely revert. Thanks!